contracts/modules/VersionManager.sol

```diff
@@ -254,10 +279,14 @@ contract VersionManager is IVersionManager, IModule, BaseFeature, Owned {
         bytes memory _data
     ) 
         external 
-        onlyFeature(_wallet)
         override
         returns (bytes memory _res) 
     {
+        require(
+            _isFeatureAuthorisedForWallet(_wallet, msg.sender) || // Wallet invoked by a feature
+            _isWalletInitializing(_wallet), // Wallet invoked by the WalletFactory
```

Comment:
This is no longer needed since we have removed the ENS logic from the `WalletFactory`.